src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -16,5 +16,8 @@ export default function (mediaFeature) {
   // SCSS or Less interpolation
   if (hasInterpolation(mediaFeature)) { return false }
 
+  // SCSS variables
+  if (/\$.+?/.test(mediaFeature)) { return false }
```

Comment:
@davidtheclark right, i create PR with this change and continue to work on this issue, 
Also in `hasInterpolation.js`:
`if (/#{.+?}|@{.+?}|\$\(.+?\)/.test(string)) { return true }`
What is `\$\(.+?\)`? `scss` and `less` not support this, i was found this in docs, bot found.